Source/JavaScriptCore/wasm/WasmOperations.cpp

```diff
@@ -71,6 +71,133 @@ namespace WasmOperationsInternal {
 }
 #endif
 
+static constexpr unsigned gprToIndex(GPRReg r)
+{
+    for (unsigned i = 0; i < GPRInfo::numberOfArgumentRegisters; ++i) {
+        if (GPRInfo::toArgumentRegister(i) == r)
+            return i;
+    }
+    RELEASE_BASSERT_NOT_REACHED();
```

Comment:
Whoops, here and immediately below, presumably shouldn't be `BASSERT` if we're not in bmalloc.

Reply:
Oh wow, oops!